src/Tribe/Tickets.php

```diff
@@ -2072,6 +2143,363 @@ public function generate_tickets_email_content( $tickets ) {
+		 *      @type array|string $headers              The list of headers to send.
+		 *      @type array        $attachments          The list of attachments to send.
+		 *      @type string       $provider             The provider slug (rsvp, tpp, woo, edd).
+		 *      @type int          $post_id              The post/event ID to send the emails for.
+		 *      @type string|int   $order_id             The order ID to send the emails for.
+		 *      @type string|int   $ticket_sent_meta_key The meta key to use for marking an attendee ticket as sent.
+		 * }
+		 *
+		 * @return bool Whether email was sent to attendees.
+		 */
+		public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
+			// If no tickets to send for, do not send email.
+			if ( empty( $tickets ) ) {
+				return false;
+			}
+
+			$defaults = [
+				'subject'              => '',
+				'content'              => '',
+				'from_name'            => '',
+				'from_email'           => '',
+				'headers'              => [],
+				'attachments'          => [],
+				'provider'             => 'ticket',
+				'post_id'              => 0,
+				'order_id'             => '',
+				'ticket_sent_meta_key' => '',
+				'send_callback'        => 'wp_mail',
+			];
+
+			// Set up the default arguments.
+			$args = array_merge( $defaults, $args );
+
+			$subject              = trim( (string) $args['subject'] );
+			$content              = trim( (string) $args['content'] );
+			$from_name            = trim( (string) $args['from_name'] );
+			$from_email           = trim( (string) $args['from_email'] );
+			$headers              = $args['headers'];
+			$attachments          = $args['attachments'];
+			$provider             = $args['provider'];
+			$post_id              = $args['post_id'];
+			$order_id             = $args['order_id'];
+			$ticket_sent_meta_key = $args['ticket_sent_meta_key'];
+			$send_callback        = $args['send_callback'];
+
+			// If invalid send callback, do not send the email.
+			if ( ! is_callable( $send_callback ) ) {
+				return false;
+			}
+
+			// Set up default content.
+			if ( empty( $content ) ) {
+				$content = $this->generate_tickets_email_content( $tickets );
+			}
+
+			// Set up default subject.
+			if ( empty( $subject ) ) {
+				$site_name = stripslashes_deep( html_entity_decode( get_bloginfo( 'name' ), ENT_QUOTES ) );
+				$is_rsvp   = 'rsvp' === $provider;
+
+				$singular = $is_rsvp
+					? tribe_get_rsvp_label_singular( 'RSVP email send' )
+					: tribe_get_ticket_label_singular_lowercase( 'ticket email send' );
+
+				$plural = $is_rsvp
+					? tribe_get_rsvp_label_plural( 'RSVPs email send' )
+					: tribe_get_ticket_label_plural_lowercase( 'tickets email send' );
+
+				// translators: %1$s: The singular of "RSVP" or "ticket", %2$s: The plural of "RSVPs" or "tickets", %3$s: The site name.
+				$subject_string = _nx( 'Your %1$s from %3$s', 'Your %2$s from %3$s', count( $tickets ), 'The default RSVP/ticket email subject', 'event-tickets' );
```

Comment:
:warning: Mismatched placeholders is probably an error

<sub>hash: eeac25acdf38836e1666bf69b5d73071</sub>